Sources/GutenbergKit/Sources/EditorViewControllerDelegate.swift

```diff
@@ -20,9 +20,94 @@ public protocol EditorViewControllerDelegate: AnyObject {
     /// - warning: This is currently also called for the initial render, which
     /// is probably not how it should be in the production design.
     func editor(_ viewController: EditorViewController, didUpdateContentWithState state: EditorState)
+
+    func editor(_ viewController: EditorViewController, didRequestMediaFromSiteMediaLibrary config: OpenMediaLibrary)
 }
 
 public struct EditorState {
     /// Set to `true` if the editor has non-empty content.
     public var isEmpty = true
 }
+
+public struct OpenMediaLibrary: Decodable {
```

Comment:
This name is a little bit unclear – could we call it `OpenMediaLibraryAction` or something?